src/Symfony/Component/Messenger/Transport/AmqpExt/Connection.php

```diff
@@ -39,6 +39,11 @@ class Connection
      */
     private $amqpQueue;
 
+    /**
+     * @var \AMQPExchange|null
+     */
+    private $amqpRetryExchange;
+
     public function __construct(array $connectionCredentials, array $exchangeConfiguration, array $queueConfiguration, bool $debug = false, AmqpFactory $amqpFactory = null)
```

Comment:
IMHO all the config keys must be described in the docblock, as in https://github.com/symfony/symfony/blob/d264fcea24cc279054c7fe480e3dafb67f694668/src/Symfony/Component/Routing/Router.php#L105-L125 or https://github.com/symfony/symfony/blob/d264fcea24cc279054c7fe480e3dafb67f694668/src/Symfony/Component/HttpFoundation/Session/Storage/NativeSessionStorage.php#L63-L97 for instance